README.md

```diff
@@ -15,6 +15,214 @@ Build
 
     $ npm run dist
 
+Usage
+-----
+
+**Caution: everything in this section is still pure fiction.**
+
+* Every operation is performed locally;
+* Synchronization with server shall be ran explicitly.
+
+### Collection
+
+```js
+
+  import Cliquetis from "cliquet";
+
+  var db = new Cliquetis();
+  var articles = db.collection('articles');
+
+```
+
+The collection object has the following attributes:
+
+* **last_modified**: last synchronization timestamp, ``undefined`` if never sync'ed.
+
+> Synchronization timestamps are persisted in the device local storage.
+
+
+### List records
+
+```js
+
+  articles.all().then(result => {
+    {
+      data: [
+        {
+          id: "705b17be-e957-4c14-8f4c-86f8eaac29c0",
+          title: "foo"
+        },
+        {
+          id: "68e63131-3859-40cc-a4f7-b237ca179329",
+          last_modified: 1432222889336,
+          title: "Web page"
+        },
+      ]
+    }
+  });
+
+```
+
+> Records with ``last_modified`` attribute were sync'ed on a server.
+
+
+### Create records
+
+```js
+
+  articles.save({title: "foo"}).then(result => {
+    data: {
+      id: "2dcd0e65-468c-4655-8015-30c8b3a1c8f8",
+      title: "foo",
+    }
+  });
+
+```
+
+> Records identifiers are generated locally using UUID4.
+
+
+### Update records
+
+```js
+
+  var existing = {
+    id: "2dcd0e65-468c-4655-8015-30c8b3a1c8f8",
+    title: "bar"
+  };
+
+  articles.save(existing).then(result => {
```

Comment:
I was thinking that in most cases we don't care, we just want to save the record.
Otherwise the ``if`` will have to be done by users ?
```
if (record.id && record.last_modified) {
    record = collection.save(record);
}
else {
    record = collection.create(record);
}
```
I would prefer to keep this logics hidden :) But I'm fully open to discussion!